upgrade.php

```diff
@@ -244,7 +244,7 @@ function getSqlServer()
     //test if mysql or mariadb is used.
     $sql = 'SELECT VERSION() as version';
     $result = $link->query($sql);
-    if (strpos($result->fetch_array['version'], 'MariaDB') === false) {
+    if (strpos($result->fetch_array()['version'], 'MariaDB') === false) {
```

Comment:
I've read somewhere here in the past that some part of the code already need 5.4.